src/app/screens/Home/components/BookList/components/Book/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import i18next from 'i18next';
+
+import { Books } from '~utils/types';
+
+import styles from './styles.module.scss';
+
+interface BookProps {
+  books: Books[];
+}
+
+export default function Book({ books }: BookProps) {
+  return (
+    <>
+      {books.map(book => (
```

Comment:
this should receive the book properties as props and dont do a map here, just render the book itself